### PR TITLE
Connect statupage API.

### PR DESCRIPTION
**Pull request by 10thfloor**

### CODE REVIEW

File: docs/content/status.mdx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+## Chains
+
+<StatusContext.Consumer>
+  {({ mainnetStatus, testnetStatus, canaryNetStatus }) => (
+    <>
+      <NetworkStatus
+        status={mainnetStatus}
+        networkName={MAINNET}
+        networkVersion="9"
```
Comment (Kay-Zee) [inline docs/content/status.mdx:26]:
```suggestion
        networkVersion="10"
```

File: docs/content/status.mdx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        versionUrl=""
+        nextSporkDate="June 23, 2021"
+      />
+      <NetworkStatus
+        status={testnetStatus}
+        networkName={TESTNET}
+        networkVersion="9"
```
Comment (Kay-Zee) [inline docs/content/status.mdx:33]:
```suggestion
        networkVersion="24"
```

if we're going by the number of sporks, then it'd be 24

File: docs/content/status.mdx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        versionUrl=""
+        nextSporkDate="June 23, 2021"
+      />
+      <NetworkStatus
+      status={canaryNetStatus}
+        networkName={CANARYNET}
+        networkVersion="9"
```
Comment (Kay-Zee) [inline docs/content/status.mdx:40]:
```suggestion
        networkVersion="6"
```

File: docs/content/status.mdx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+<StatusContext.Consumer>
+  {({ mainnetStatus, testnetStatus, canaryNetStatus }) => (
+    <>
+      <NetworkStatus
+        status={mainnetStatus}
+        networkName={MAINNET}
+        networkVersion="9"
+        versionUrl=""
+        nextSporkDate="June 23, 2021"
```
Comment (Kay-Zee) [inline docs/content/status.mdx:28]: These will need to be updated as well, before merge

File: docs/content/status.mdx [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+      />
+      <NetworkStatus
+        status={testnetStatus}
+        networkName={TESTNET}
+        networkVersion="9"
+        versionUrl=""
+        nextSporkDate="June 23, 2021"
```
Comment (Kay-Zee) [inline docs/content/status.mdx:35]: These will need to be updated as well, before merge

File: docs/content/status.mdx [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+      />
+      <NetworkStatus
+      status={canaryNetStatus}
+        networkName={CANARYNET}
+        networkVersion="9"
+        versionUrl=""
+        nextSporkDate="June 23, 2021"
```
Comment (Kay-Zee) [inline docs/content/status.mdx:42]: These will need to be updated as well, before merge

**Review by Kay-Zee — COMMENTED** [review]
I noticed that the colour starts off as red when the page first loads. Would it be possible to add an "Unknown" state that's grey, and start off as that? The red, although very brief is somewhat alarming.

File: docs/plugins/gatsby-theme-flow/src/components/flow-status/context.js [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+    (url) =>
+      fetch(url, {
+        headers: {
+          Authorization: `OAuth ${process.env.GATSBY_STATUSPAGE_API_KEY}`
+        }
+      }).then((r) => r.json()),
+    {
+      refreshInterval: 100000,
+      onSuccess: (result) => {
+        console.log(result);
```
Comment (10thfloor) [inline docs/plugins/gatsby-theme-flow/src/components/flow-status/context.js:31]:
```suggestion
```

**Pull request merged**